runtime-js/numbers.js

```diff
@@ -216,6 +216,8 @@ atr$(JSNum$proto, 'character', function(){
   }
   return Character(this.valueOf());
 },undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','character']};});
+atr$(JSNum$proto, 'nearestFloat', function(){ return Float(this.valueOf()); },
+  undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','nearestFloat']};});
```

Comment:
Hmm, it doesn't seem to work – adding the `Integer/nearestFloat.js` file seems to have no effect at all. If I also remove the lines above from numbers.js, I just get this error:
```
test.js.quick:
[ceylon-run-js] Equality, Comparable, clones
[ceylon-run-js] Booleans, Numbers, Bytes, Characters, Strings
[ceylon-run-js] 
[ceylon-run-js] /home/paulo/projektoj/ceylon.language/build/test/default/default.js:7212
[ceylon-run-js] check((2).nearestFloat.equals(m$1.Float(2.0)),"small integer nearestFloat");
[ceylon-run-js]                        ^
[ceylon-run-js] TypeError: Cannot call method 'equals' of undefined
[ceylon-run-js]     at numbers (/home/paulo/projektoj/ceylon.language/build/test/default/default.js:7212:24)
[ceylon-run-js]     at run (/home/paulo/projektoj/ceylon.language/build/test/default/default.js:8499:1)
[ceylon-run-js]     at [eval]:1:267
[ceylon-run-js]     at Object.<anonymous> ([eval]-wrapper:6:22)
[ceylon-run-js]     at Module._compile (module.js:456:26)
[ceylon-run-js]     at evalScript (node.js:532:25)
[ceylon-run-js]     at startup (node.js:80:7)
[ceylon-run-js]     at node.js:902:3
```
Looking at the `ceylon.language-1.1.1.js` (in build/dist/repo/...), I also don't see any mention of `nearestFloat` in there.

It also looks like the Stitcher doesn't consider the native methods from Integer.ceylon at all, since Integer.ceylon is excluded:
```
ceylon.language.js:
     [java] Generating language module compile-time model in JSON...
     [java] Sep 20, 2015 6:22:17 PM com.redhat.ceylon.common.log.JULLogger infoCompiling language module sources: core_functions.js, is-op.js, exists-nonempty.js, type_arithmetic.js, -Anything, Anything.js, Object, Null, Identifiable, Callable, callables.js, dynamic_functions.js
     [java] 
     [java] INFORMATION: [signing jar /home/paulo/projektoj/ceylon.language/build/dist/repo/ceylon/language/1.1.1/ceylon.language-1.1.1-model.js into: /home/paulo/projektoj/ceylon.language/build/dist/repo/ceylon/language/1.1.1/ceylon.language-1.1.1-model.js.sha1]
     [java] EXCLUDING ../ceylon.language/src/ceylon/language/Anything.ceylon
     [java] Compiling language module sources: Basic, Throwable, Exception, Comparison, Iterable, comprehensions.js, -Boolean, Boolean.js, Tuple, tuples.js, impl/
     [java] EXCLUDING ../ceylon.language/src/ceylon/language/Boolean.ceylon
     [java] Stitching in ../ceylon.language/runtime-js/Throwable_cons_before.js. It must contain an anonymous function which will be invoked with the same arguments as the Throwable constructor.
     [java] Compiling language module sources: Numeric, Integral, Scalable, Number, -Integer, -Float, numbers.js, Array, sequences.js, String, strings.js
     [java] EXCLUDING ../ceylon.language/src/ceylon/language/Integer.ceylon
     [java] EXCLUDING ../ceylon.language/src/ceylon/language/Float.ceylon
     [java] Stitching in native method ceylon.language::String.compare, ignoring Ceylon definition
     [java] Stitching in native method ceylon.language::String.equals, ignoring Ceylon definition
     [java] Compiling language module sources: process.js
     [java] Compiling language module sources: .
```
Do I have to do something else to make the Stitcher consider Integer's native methods, or does this just not work yet?